validator/keymanager/remote_wallet.go

```diff
@@ -0,0 +1,128 @@
+package keymanager
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	RW "github.com/alonmuroch/validatorremotewallet/wallet/v1alpha1"
+	ethpb "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
+	"github.com/prysmaticlabs/prysm/shared/bls"
+	"github.com/prysmaticlabs/prysm/shared/bytesutil"
+	grpc "google.golang.org/grpc"
+	"time"
+)
+
+type RemotewalletOpts struct {
+	Url    string   `json:"url"`
+}
+
+// TODO
+var RemotewalletOptsHelp = `The remote wallet key manager stores keys in a remote store.  The options are:
+  - location This is the location to look for wallets.  If not supplied it will
+    use the standard (operating system-dependent) path.
+
+A remote wallet key manager service example here:  
+
+A sample remote keymanager options file is:
+
+  {
+    "url":    "<url for your remote wallet service>",  
+  }`
+
+func NewRemoteWallet(input string) (KeyManager, string, error) {
+	// parse config file
+	opts := &RemotewalletOpts{}
+	err := json.Unmarshal([]byte(input), opts)
+	if err != nil {
+		return nil, walletOptsHelp, err
+	}
+
+	// try and connect to remote wallet
+	conn, err := grpc.Dial(opts.Url, grpc.WithInsecure(), grpc.WithBlock())
+	if err != nil {
+		log.Fatalf("did not connect to remote wallet: %v", err)
+	} else {
+		log.Printf("connected to remote wallet at: %v", opts.Url)
+	}
+	//defer conn.Close() TODO
+	c := RW.NewRemoteWalletClient(conn)
+
+	km := &RemoteWallet{
+		keys: make([][48]byte,0),
```

Comment:
can we define `48` as a constant somewhere?

Reply:
I was thinking that as well. It makes a lot of sense. 
I'll try and do a different PR for that